### ISSUE TITLE
Settings Moved Popup

### ISSUE THREAD

**Issue by jabacchetta** [Gruntfuggly/todo-tree#211]
This popup appears to keep showing with every reload, even if the settings have been updated. I suppose the obvious fix is to just click the "don't show this again" button, but I'd imagine this shouldn't be happening?

<img width="1280" alt="Screen Shot 2019-09-11 at 2 45 55 PM" src="https://user-images.githubusercontent.com/12591890/64729593-13e45680-d4a3-11e9-97e4-d1bd23227c9a.png">

**Comment by aldebaran798** [comment]
Same issue for me :/

**Comment by 181** [comment]
Same here

**Comment by crystalfp** [comment]
The update does not move the `tags` setting. Fixed this and removed the old settings (the settings editor downlight (?) the unused entries) makes the message disappear.

**Comment by jabacchetta** [comment]
Yeah, I did manually update that setting, but the notification continued for some reason. But, now it appears to have stopped. I'm not sure why... maybe I had to reload or I might not have been on the latest version that got added today.

Will leave it open in case others are experiencing the issue still. @Gruntfuggly Feel free to close this  as you'd like.

**Comment by Gruntfuggly** [comment]
Version 0.0.149 didn't save the 'don't show this again' setting in the right place so you may have seen it multple times. 0.0.151 should now work as expected. Sorry for the confusion!

It seems like it doesn't detect that the tags setting is an array properly (I thought I had tested that) so it's not being moved - will fix.
